protostar/starknet/cheatable_cairo_function_runner.py

```diff
@@ -0,0 +1,14 @@
+from typing import Optional, Dict, Any
+from starkware.cairo.common.cairo_function_runner import CairoFunctionRunner
+from protostar.starknet.cheatable_cairo_vm import CheatableVirtualMachine
+
+
+class CheatableCairoFunctionRunner(CairoFunctionRunner):
+    # MODIFICATION vm_class=VirutalMachine -> vm_class=CheatableVirtualMachine # TODO
```

Comment:
You can describe the class using triple double-quotes (see: ForkableStarknet for a reference)
`
"""
"""
`